src/AppInstallerCommonCore/Manifest/YamlWriter.cpp

```diff
@@ -0,0 +1,661 @@
+// Copyright (c) Microsoft Corporation.
+// Licensed under the MIT License.
+#include "pch.h"
+#include "AppInstallerStrings.h"
+#include "AppInstallerSHA256.h"
+#include "winget/Yaml.h"
+#include "winget/ManifestYamlWriter.h"
+
+namespace AppInstaller::Manifest::YamlWriter
+{
+    using namespace Utility::literals;
+
+    namespace
+    {
+        constexpr Utility::LocIndView PackageIdentifier = "PackageIdentifier"_liv;
```

Comment:
nit: we are not dealing with outputting to user here, soLocIndView is not needed just string_view is enough